tasty-inspector/src/scala/tasty/inspector/TastyInspector.scala

```diff
@@ -65,31 +58,27 @@ trait TastyInspector:
    *  @param classes List of paths of `.tasty` and `.jar` files (no validation is performed)
    *  @param classpath Classpath with extra dependencies needed to load class in the `.tasty` files
    */
-  protected[inspector] def inspectFilesInContext(classpath: List[String], classes: List[String])(using Context): Unit =
-    if (classes.isEmpty) report.error("Parameter classes should no be empty")
-    inspectorDriver().process(inspectorArgs(classpath, classes), summon[Context])
+  // protected[inspector] def inspectFilesInContext(classpath: List[String], classes: List[String])(using Context): Unit =
```

Comment:
We absolutely need this method in the doctool, otherwise we cannot properly forward errors to SBT. Under the visitor approach, it could be exposed in an object in `scala.tasty.inspector.internal` package.